stripper/ze_touhou_gensokyo_o4.cfg

```diff
@@ -1,3 +1,51 @@
+;Make getting a custom CT model reset player color (doesnt apply to mapper's custom models)
+modify:
+{
+	match:
+	{
+		"targetname" "rmbut"
+		"classname" "trigger_multiple"
+	}
+	insert:
+	{
+		"OnStartTouch" "!activator,color,255 255 255,0,-1"
+	}
+}
+modify:
+{
+	match:
+	{
+		"targetname" "rmbut2"
+		"classname" "trigger_multiple"
+	}
+	insert:
+	{
+		"OnStartTouch" "!activator,color,255 255 255,0,-1"
+	}
+}
+
+;Give server settings priority
+;Remove message about chat sounds, since it was removed when mapper backdoor was removed
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+	}
+	delete:
+	{
+		"OnMapSpawn" "cmdCommandmp_freezetime 201"
+		"OnMapSpawn" "cmdCommandsay **tips:humans can type words in chat to play funny sounds,there are display on board**101"
+	}
+}
+
+;Remove messages on board about chat sounds, since they were removed along with the mapper backdoor
+filter:
+{
+	"classname" "point_worldtext"
+	"targetname" "pianowords"
+}
```

Comment:
I'd rather see say sounds actually fixed rather than just making people forget about the bug? Meant to do this at some point but you can do it if you'd like.

Reply:
Honestly, I hate the say sounds (expecially since, at least on other servers, !music doesnt affect them), so I view them not being in as a benefit. I'll remove the filter for the text though, since you want them back in at some point